lib/allscripts_api/named_magic_methods.rb

```diff
@@ -132,5 +132,24 @@ def get_patient_problems(patient_id,
       results = magic("GetPatientProblems", magic_params: params)
       results["getpatientproblemsinfo"]
     end
+
+    # a wrapper around GetSchedule, returns appointments scheduled under the
+    # the user for a given date range
+    #
+    # @param start_date [Date] start date inclusive
+    # @param end_date [Date] end date inclusive
+    # @return [Array<Hash>, Array, MagicError] a list of scheduled appointments,
+    # an empty array, or an error
+    def get_appointments(start_date, end_date)
+      date_range_formatted =
+        "#{start_date.strftime('%m/%d/%Y')}|#{end_date.strftime('%m/%d/%Y')}"
```

Comment:
I wonder if ` "#{start_date.strftime('%m/%d/%Y')}|#{end_date.strftime('%m/%d/%Y')}"` should be its own method?